python/mxnet/module/base_module.py

```diff
@@ -399,7 +399,7 @@ def fit(self, train_data, eval_data=None, eval_metric='acc',
             eval_batch_end_callback=None, initializer=Uniform(0.01),
             arg_params=None, aux_params=None, allow_missing=False,
             force_rebind=False, force_init=False, begin_epoch=0, num_epoch=None,
-            validation_metric=None, monitor=None, sparse_row_id_fn=None):
+            validation_metric=None, monitor=None, sparse_row_id_fn=None, epoch_size=None):
```

Comment:
fit already has far too many arguments. Try using resizeiter

Reply:
But we would still need to know the size of data iterator to determine number of batches to resize to

Reply:
Oh you mean just relying on the user to use resizeIter?